src/lib/Microsoft.Fx.Portability.Cci/TypeExtensions.cs

```diff
@@ -609,7 +609,7 @@ public static bool IsReferenceAssembly(this IAssembly assembly)
 
         public static string MapPublicKeyTokenToName(string publicKeyToken)
         {
-            if (!TokenNames.TryGetValue(publicKeyToken.ToLower(), out var name))
+            if (!TokenNames.TryGetValue(publicKeyToken.ToLowerInvariant(), out var name))
```

Comment:
Not a blocker for this, but the `TokenNames` field should probably be created with a `StringComparer.OrdinalIgnoreCase` comparer so no string manipulations are needed

Reply:
good point! Fixed

Reply:
For the record, I don't see this fix in the PR.